modules/distr-tracing-rn-new-features.adoc

```diff
@@ -0,0 +1,38 @@
+////
+Module included in the following assemblies:
+* distr-tracing--release-notes.adoc
+////
+////
+Feature – Describe the new functionality available to the customer. For enhancements, try to describe as specifically as possible where the customer will see changes.
+Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.). For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.
+Result – If changed, describe the current user experience.
+////
+
+[id="distr-tracing-rn-new-features_{context}"]
+== New features and enhancements {ProductName} 2.0.0
+
+This release marks the rebranding of Red Hat OpenShift Jaeger to {ProductName}.  This effort includes the following:
+
+* Updates {ProductShortName} Operator to Jaeger 1.28.  Going forward, {ProductName} will only support the `stable` Operator channel. Channels for individual releases are no longer supported.
```

Comment:
this is an awesome bullet list! Much easier to scan and quickly comprehend. Great revisions!